### PR TITLE
google maps infowindow - disable autoPan

### PR DESCRIPTION
**Pull request by trickpattyFH20**
## Description

significant usability issue fixed for **mobile devices**

## Motivation and Context

(on mobile devices) when moving the map around, or resizing the map, the `infoWindow` pops up over different items on the map, causing the map to jump around a lot or "`autoPan`."

There is a setting in the google maps API on maps.infoWindow that can be passed to fix this:
`disableAutoPan: true`

## How Has This Been Tested?
I added this functionality to my build and tested on an iPhone 6 and Android with the server running on an AWS ec2 instance and found improved usability.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by frdmn** [comment]
maybe relevant to #1525

**Comment by trickpattyFH20** [comment]
related, but this a better solution than totally disabling infoWindow.

disabling the infoWindow on mobile would mean that users could not see the despawn timers :/ and that is one of the main features of the application. 

I would be much more in favor of just preventing the autoPan

File: static/map.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function initMap() {
     google.maps.event.addListenerOnce(map, 'idle', function(){
         updateMap();
     });
-    
+
```
Comment (Chlodochar) [inline static/map.js]: Unrelated. Please remove.

File: static/map.js [hunk 2]
```diff
@@ -499,7 +502,7 @@ function updateMap() {
         type: 'GET',
         data: {
             'pokemon': loadPokemon,
-            'pokestops': loadPokestops, 
+            'pokestops': loadPokestops,
```
Comment (Chlodochar) [inline static/map.js]: Unrelated. Please remove.

File: static/map.js [hunk 3]
```diff
@@ -914,7 +918,7 @@ $(function () {
             map_pokemons = {}
         }
     });
-    
+
```
Comment (Chlodochar) [inline static/map.js]: Unrelated. Please remove.

File: static/map.js (hunk 2 above)
Comment (trickpattyFH20) [inline static/map.js]: this change removes unnecessary whitespace, can it stay?

File: static/map.js (hunk 3 above)
Comment (trickpattyFH20) [inline static/map.js]: this change removes unnecessary whitespace, can it stay?

**Comment by MonkeyLeeT** [comment]
Nice fix! Merged.
